src/app/public/modules/input-box/input-box.component.ts

```diff
@@ -95,16 +97,21 @@ export class SkyInputBoxComponent implements OnInit {
   }
 
   public formControlFocusIn(): void {
-    this.formControlHasFocus = true;
+    setTimeout(() => {
```

Comment:
We've made a point in the past to add comments above any `setTimeout` functions so that future engineers know why they're there and don't remove them inadvertently. Do you mind adding a comment or two?

Reply:
I was meaning to circle back on that earlier. I hit an error that `setTimeout()` fixed, but I ended up refactoring so much after that that it might not be needed. I removed it for now.